index.html

```diff
@@ -1583,23 +1641,51 @@ <h5 id="Arcane Traits">Arcane Traits</h5>
                <td>Long Range Teleport
                </td>
                <td>
-                  <span class="wip">
-                     Maybe worth adding for other campaigns but I won’t use it for mine. 10
-                     minutes to cast a multi-person teleport to a known location within X miles.
-                  </span>
+                  This is an optional ability that is not used by default in all campaigns. It 
+                  is not a part of the Arcus lore but GM's may still wish to use it for other 
+                  campaigns.
+                  <p>
+                  You may spend an hour creating a magical symbol on the ground that is 10ft in diameter. 
+                  On completion this opens a gateway to another location that you have visited before. 
```

Comment:
Generally, a lot more fond of the portal location to location teleports than just a long range teleport where you have to know the "address" (or symbols).  This is a) less OP b) basically already exists within The Spires of Arcus with what the soul evil dude did.